exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "TestSingleBookPrice",
```

Comment:
previous discussion https://github.com/exercism/x-common/pull/465#discussion_r91590493

> In retrospect this probably isn't needed. I have a property that just returns the price of a single book. The test is just confirming that it equals 8.

I would agree. Not every implementation necessarliy needs to have this property in order to be correct. We would [prefer not to drive the implementation](https://github.com/exercism/discussions/issues/41).